<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtComMate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="SerialReceiveGroupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>790</width>
      <height>330</height>
     </rect>
    </property>
    <property name="title">
     <string>串口接收设置</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>168</width>
       <height>300</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="SerialReceiveFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="SerialComLabel">
        <property name="text">
         <string>COM</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="SerialPort_ComBoBox" name="SerialComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="SerialBaudrateLabel">
        <property name="text">
         <string>波特率</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="SerialBaudrateComboBox">
        <item>
         <property name="text">
          <string>custom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>110</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>300</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>76800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>230400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>256000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>460800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>512000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>576000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>921600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1500000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2000000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="SerialStopBitLabel">
        <property name="text">
         <string>停止位</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="SerialStopBitComboBox">
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="SerialDataBitLabel">
        <property name="text">
         <string>数据位</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="SerialDataBitcomboBox">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="SerialChecksumBitLabel">
        <property name="text">
         <string>校验位</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="SerialChecksumBitComboBox">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Even</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Odd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mark</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Space</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="SerialReceiveHexCheckBox">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="SerialReceiveTimestampCheckBox">
        <property name="text">
         <string>时间戳</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="SerialReceiveClearPushButton">
        <property name="text">
         <string>清空接收区</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="SerialConnectComPushButton">
        <property name="text">
         <string>连接COM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="SerialReceiveTextEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>591</width>
       <height>290</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SerialSendGroupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>340</y>
      <width>790</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>串口发送设置</string>
    </property>
    <widget class="QTextEdit" name="SerialSendTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>641</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="SerialSendRepeatGroupBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>重复</string>
     </property>
     <widget class="QCheckBox" name="SerialSendRepeatCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="SerialSendRepeatDurationLineEdit">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>25</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="SerialSendRepeatUnitLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>23</y>
        <width>25</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ms</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="SerialSendPushButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>90</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="SerialHardFlowControlGroupBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>硬件流控</string>
     </property>
     <widget class="QCheckBox" name="SerialHardFlowControlDSRDTRCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>dsr/dtr</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="SerialHardFlowControlRTSCTSCheckBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>25</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>rts/cts</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="SerialSendLineFeedGroupBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>换行</string>
     </property>
     <widget class="QComboBox" name="SerialSendLineFeedComboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>/r/n</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>/n</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QCheckBox" name="SerialSendHexCheckBox">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hex</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="SerialSoftFlowControlGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>软件流控</string>
     </property>
     <widget class="QCheckBox" name="SerialSoftFlowControlCheckBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>25</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>xon/xoff</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="SerialSendClearPushButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>120</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="SerialStatusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="MenuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="CnLanguageAction"/>
     <addaction name="EnLanguageAction"/>
    </widget>
    <addaction name="MenuLanguage"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="CnLanguageAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>zh-CN</string>
   </property>
  </action>
  <action name="EnLanguageAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>en</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SerialPort_ComBoBox</class>
   <extends>QComboBox</extends>
   <header>serialport_combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
